runtime/vm/classsupport.c

```diff
@@ -1093,6 +1093,9 @@ loadWarmClassFromSnapshot(J9VMThread *vmThread, J9ClassLoader *classLoader, J9Cl
 				if (NULL != moduleObject) {
 					J9VMJAVALANGCLASS_SET_MODULE(vmThread, classObject, moduleObject);
 				}
+			} else {
+				/* unnamed module */
+				J9VMJAVALANGCLASS_SET_MODULE(vmThread, classObject, J9VMJAVALANGCLASSLOADER_UNNAMEDMODULE(vmThread, clazz->classLoader->classLoaderObject));
```

Comment:
Just for confirmation: is `classLoaderObject` always expected to be non-null in this code path?